### ISSUE TITLE
[Request]: Minimum dataset size needed for good trained model performance?

### ISSUE THREAD

**Issue by aubricot** [aubricot/nsm#4]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you would like.

I'm curious what the minimum training dataset size and diversity are to achieve satisfactory model performance. I would like a table containing information from training experiments using downsampled versions of your dataset and resulting model accuracy.

### Describe alternatives you have considered.

N/A

### Filename

train_model.py and README.md

### Version

python 3.11.13  pytorch 2.5.1  pytorch-cuda 12.4

### Any log output or other content that may be useful for developing the requested feature.

```shell

```
